resource/validate.go

```diff
@@ -31,6 +31,7 @@ const (
 
 type TestResult struct {
 	Successful   bool          `json:"successful" yaml:"successful"`
+	Skipped      bool          `json:"skipped" yaml:"skipped"`
```

Comment:
This was an acidental add from some left over previous work, however I think it's useful as atm only way to find out if it was skipped is by the int value from `Result` and thats quite hard to use when looking at JSON representations of results.